Add PR description template (#1)
3778/.github: PULL_REQUEST_TEMPLATE.md

> Thank you for contributing, please fill these checks and give more information about the changes you're introducing.
> Many of these questions are optional, but please keep in mind that a better description leads to a better review.
> For questions that don't apply to your changes, just answer `N/A`.
> 
> If you need the help of any person or team to apply these changes, please `@` them in the end of the description or mark them as reviewers, if it makes more sense.

Reviewer: ```suggestion
<!---
Thank you for contributing, please fill these checks and give more information about the changes you're introducing.
Many of these questions are optional, but please keep in mind that a better description leads to a better review.
For questions that don't apply to your changes, just answer `N/A`.

If you need the help of any person or team to apply these changes, please `@` them in the end of the description or mark them as reviewers, if it makes more sense.
--->
```
Using it as a comment shows it to the person opening the PR but hides it from everyone else when rendering.

> Thank you for contributing, please fill these checks and give more information about the changes you're introducing.
> Many of these questions are optional, but please keep in mind that a better description leads to a better review.
> For questions that don't apply to your changes, just answer `N/A`.
> 
> If you need the help of any person or team to apply these changes, please `@` them in the end of the description or mark them as reviewers, if it makes more sense.

Reviewer: ```suggestion
<!---
Thank you for contributing, please fill these checks and give more information about the changes you're introducing.
Many of these questions are optional, but please keep in mind that a better description leads to a better review.
For questions that don't apply to your changes, just answer `N/A`.

If you need the help of any person or team to apply these changes, please `@` them in the end of the description or mark them as reviewers, if it makes more sense.
--->
```

Using it as a comment shows it to the person opening the PR but hides it from everyone else when rendering.

#### Plese check if the pull request fulfills these requirements:

> - [ ] The commit message follows our [guidelines](https://3778.atlassian.net/wiki/spaces/ENG/pages/206897153/Git+GitHub#Commit-Message);

Reviewer: ```suggestion
- [ ] The commit message follows our [guidelines](https://3778.atlassian.net/wiki/spaces/ENG/pages/252543089/How+We+Use+GitHub);
```

Added a link to the new related page

Thank you for contributing, please fill these checks and give more information about the changes you're introducing.

> Many of these questions are optional, but please have in mind that a better description lead to a better review.

Reviewer: ```suggestion
Many of these questions are optional, but please keep in mind that a better description leads to a better review.
```

For questions that don't apply to your changes, just answer `N/A`.

> If you need the help of any person or team to apply these changes, please address to them in the end of the description or mark them as reviwers, if it makes more sense.

Reviewer: ```suggestion
If you need the help of any person or team to apply these changes, please `@` them in the end of the description or mark them as reviewers, if it makes more sense.
```

* Plese check if the pull request fulfills these requirements:

> - [ ] The commit message follows our guidelines;

Reviewer: maybe link to guidelines?

Author: It would be better, but we don't have a guideline yet. It is one of the objectives of the git policy we are building. I wrote it here as a provocation or reminder, do you think is better to keep it off the template until we have proper guidelines?

- [ ] Tests for the changes have been added/performed;
- [ ] Docs have been added/updated.

* What's the purpose of this pull request?

* If it is a bug fix, what was the root cause?

* What did you do to accomplish its purpose? Explain what you've done.

* Does this pull request introduce breaking changes?

* How to test this feature/fix?

> * There is anyone more suited to test the changes introduced by this pull request?

Reviewer: ```suggestion
* Is anyone more suited to test the changes introduced by this pull request?
```
